src/encrypt.ts

```diff
@@ -0,0 +1,11 @@
+import { encryptString } from "@govtechsg/oa-encryption";
+import fs from "fs";
+import signale from "signale";
+import chalk from "chalk";
+
+export const encrypt = (input: string, output: string): void => {
```

Comment:
might want to do path validation on input and output so that this command isn't a directory traversal vulnerability

Reply:
#22